### PR TITLE
feat: add documentation pages

### PR DESCRIPTION
**Pull request by shaik-saad**
This pr includes
- Pages for all p0 and p1 components
- Pages for utility classes

### CODE REVIEW

**Review by NamrataRaikwar2002 — APPROVED** [review]

**Comment by kart-c** [comment]
A few suggestions I have saad

- You can add `cursor: pointer` to the menu toggle icon which opens up in the tablet/mobile view
- The side navigation - when a user hovers on the `li` list item the background color changes. This made me think the item is clickable, but user has to actually click the `a` tag, the actual word inside to navigate. My opinion is to make the `a` tag the size of `li` so the user can easily navigate your website.

Other than this, the UI is clean and simple.

File: documentation/components/badge.html [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+                    <figure class="avatar avatar-lg badge-container">
+                        <img src="../../assets/avatar.jpg" alt=""></img>
+                        <span class="badge avatar-badge"></span>
+                    </figure>
+                    <figure class="avatar avatar-md badge-container">
+                        <img src="../../assets/avatar.jpg" alt=""></img>
+                        <span class="badge avatar-badge"></span>
+                    </figure>
+                    <figure class="avatar avatar-sm badge-container">
+                        <img src="../../assets/avatar.jpg" alt=""></img>
```
Comment (kart-c) [inline documentation/components/badge.html]: Add `alt` attributes, it helps in accessibility

**Comment by shaik-saad** [comment]
> A few suggestions I have saad
> 
> * You can add `cursor: pointer` to the menu toggle icon which opens up in the tablet/mobile view
> * The side navigation - when a user hovers on the `li` list item the background color changes. This made me think the item is clickable, but user has to actually click the `a` tag, the actual word inside to navigate. My opinion is to make the `a` tag the size of `li` so the user can easily navigate your website.
> 
> Other than this, the UI is clean and simple.

File: documentation/components/badge.html (hunk 1 above)
Comment (shaik-saad) [reply documentation/components/badge.html]: Okay sure.

**Comment by shaik-saad** [comment]
Thanks for the suggestions @kart-c Kartik, they helped alot.

**Review by kart-c — APPROVED** [review]

**Pull request merged**